src/pages/DemoSetupPage.js

```diff
@@ -18,12 +20,16 @@ const propTypes = {
  * route that led the user here. Now, it's just used to route the user home so we
  * don't show them a "Hmm... It's not here" message (which looks broken).
  */
-function DemoSetupPage() {
-    useFocusEffect(() => {
-        Navigation.isNavigationReady().then(() => {
-            Navigation.goBack(ROUTES.HOME);
-        });
-    });
+function DemoSetupPage(props) {
+    useFocusEffect(
+        useCallback(() => {
```

Comment:
NAB. But is there any specific reason why we are no longer using the navigation promise?

Reply:
I looked around at other similar implementations and noticed we don't use it anymore - and we were somehow getting inside this call twice which called `CreateChatReport` twice, but without that function we only get there once 🤷